src/views/v2/month/calendar-body/day/multiday-events/multiday-event/bar.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * View: Month View - Multiday Event Event Bar
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/month/calendar-body/day/multiday-events/multiday-event/bar.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @since TBD
+ *
+ * @var boolean $should_display If the event starts today and this week.
+ * @var string $grid_start_date The `Y-m-d` date of the day where the grid starts.
+ * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
+ *
+ * @see tribe_get_event() For the format of the event object.
+ *
+ * @version TBD
+ */
+?>
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: e2316898b6a51aa0e01b44313d64149c</sub>